access/api.go

```diff
@@ -52,6 +53,10 @@ type API interface {
 
 	GetExecutionResultForBlockID(ctx context.Context, blockID flow.Identifier) (*flow.ExecutionResult, error)
 	GetExecutionResultByID(ctx context.Context, id flow.Identifier) (*flow.ExecutionResult, error)
+
+	// SubscribeBlocks subscribes to the blocks starting from a specific block ID and block height, with an optional block status.
+	// By default, returns sealed blocks.
```

Comment:
A bit confused by "default", we always need to provide a value `blockStatus` so there is no default, I would suggest to remove it. If caller passes `blockStatus == Unknown` we should bail out with error.